### DISCUSSION TITLE
Thanks and Suggestions

### DISCUSSION THREAD

**Discussion by Xylemflo** [ZL-Audio/ZLEqualizer#90] [General]
Hello ZL audio and first of all, thank you for the time and effort you have put into developing this EQ. It has quickly become a favorite of all the free and paid EQs that I use. One feature that I struggle with is the popup/hover box above each node on the EQ. Here are a few suggestions/requests for development. 

1. Not being able to see the frequency and the gain value in this box is counterintuitive for me. I am almost never trying to EQ by note value, and I always want to know the gain that I am adding/subtracting. An option to set a default here to see freq and gain in db would be great. An alternative would be to make the information in the lower panel ( freq, gain, Q) always on in real time. 
2. This is a smaller issue, but the on/off button for mute gets mistaken for an on/off button for the node. IMHO, it is more intuitive that a lit icon would have the node on (default), and turning it off would be mute (eg: the button and the node goes gray/dim). 

I hope you consider these to be constructive comments, and thanks again for developing this.

**Comment by zsliu98** [comment]
> Not being able to see the frequency and the gain value in this box is counterintuitive for me. I am almost never trying to EQ by note value, and I always want to know the gain that I am adding/subtracting. An option to set a default here to see freq and gain in db would be great. An alternative would be to make the information in the lower panel ( freq, gain, Q) always on in real time.

Please refer to #87. I *may* add an option to show freq/gain instead of pitch note (don't take it as a granted).

> This is a smaller issue, but the on/off button for mute gets mistaken for an on/off button for the node. IMHO, it is more intuitive that a lit icon would have the node on (default), and turning it off would be mute (eg: the button and the node goes gray/dim).

You are correct. I am about to change that (and some other UI staff) in the next minor/patch version.

**Comment by nicktshredz** [comment]
Huge fan of this plug-in, thank you so much for making it! Alas, I have 2 requests!
1. Could you make it where when you hover the cursor over the buttons on the bottom panel the functions appear?
The dynamic parameters are a little confusing ('s' for side chain makes sense but I do not know what the arrow above the side-chain button does lol)
2. Would you be able to make dynamic linking 'on' by default instead of 'off', I know there's a huge advantage to being un-linked but it is a little tedious to turn it on every time. I have a default preset where it starts out ON, which can alleviate this; but it is a little confusing turning on dynamic mode and the moment you move the band the dynamic info doesn't follow. Maybe something you could save in the UI section?

Thank you again for this amazing plugin

**Reply by zsliu98** [reply]
@nicktshredz 

1. I don't like tooltips. When you press the arrow button, the dynamic is in the relative mode. It will calculate the loudness difference between the trigger band and the whole side-chain, and compare it with the threshold.
2. Your approach is correct (i.e., saving `Default Dynamic Link` on as the default preset). I may move it to the UI section.

Please open another discussion if you have any further suggestions.
